src/tests/test_rng.cpp

```diff
@@ -771,6 +771,21 @@ class System_RNG_Tests final : public Test
             rng.add_entropy(out_buf.data(), out_buf.size());
             }
 
+#if (BOTAN_MP_WORD_BITS == 64)
+         // Pass buffer with a size greater than 32bit
+         const size_t size32BitsMax = std::numeric_limits<uint32_t>::max();
+         const size_t checkSize = 1024;
+         std::vector<uint8_t> large_buf(size32BitsMax + checkSize);
+         std::memset(large_buf.data() + size32BitsMax, 0xfe, checkSize);
+
+         rng.randomize(large_buf.data(), large_buf.size());
+
+         std::vector<uint8_t> check_buf(checkSize);
+         std::memset(check_buf.data(), 0xfe, checkSize);
```

Comment:
This could just be `std::vector<uint8_t> check_buf(checkSize, 0xFE)`

Reply:
True!